runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -223,6 +219,11 @@ public void onVarbitChanged(VarbitChanged event)
 		int imbuedHeartCooldownVarb = client.getVarbitValue(Varbits.IMBUED_HEART_COOLDOWN);
 		int homeTeleportVarp = client.getVar(VarPlayer.LAST_HOME_TELEPORT);
 		int minigameTeleportVarp = client.getVar(VarPlayer.LAST_MINIGAME_TELEPORT);
+		int staminaEffectActive = client.getVarbitValue(Varbits.RUN_SLOWED_DEPLETION_ACTIVE);
```

Comment:
Is there ever a case where this would be `0`, but `lastStaminaEffect` and `totalStaminaEffect` would still be the same value? I don't think we need to check this--I believe `(lastStaminaEffect != totalStaminaEffect && config.showStamina())` is all that's needed to check for stamina boost changes.

Reply:
I can't think of any logical case where that would happen, but it's checked here: https://github.com/Joshua-F/cs2-scripts/blob/741271f0c3395048c1bad4af7881a13734516adf/scripts/%5Bproc%2Cbuff_bar_get_value%5D.cs2#L25
So I'd assume there might be a special case where this would happen... or it's just jagex being jagex...
Up to you, If you think it needs to be removed, it's gone.